### PR TITLE
Do not log trailing whitespace

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
Follow [https://github.com/kubernetes/kubernetes/pull/56409](https://github.com/kubernetes/kubernetes/pull/56409)

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by stewart-yu** [comment]
cc @deads2k Please help review this pr, thanks!

File: pkg/cloudprovider/providers/azure/azure_blobDiskController.go [hunk 1]
```diff
@@ -535,7 +535,7 @@ func (c *BlobDiskController) createStorageAccount(storageAccountName string, sto
 			return fmt.Errorf("azureDisk - can not create new storage account, current storage accounts count:%v Max is:%v", len(c.accounts), maxStorageAccounts)
 		}
 
-		glog.V(2).Infof("azureDisk - Creating storage account %s type %s \n", storageAccountName, string(storageAccountType))
+		glog.V(2).Infof("azureDisk - Creating storage account %s type %s\n", storageAccountName, string(storageAccountType))
```
Comment (deads2k) [inline pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: no `\n` on glogs.

Comment (stewart-yu) [reply pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: Done, thanks.

**Pull request merged**